.github/workflows/pnp-input-api-build.yml

```diff
@@ -67,7 +67,17 @@ on:
         type: boolean
         required: false
         default: false
-        
+      service-name:
+          description: |
+            The name of the service that will be used for generating WID token.
+          type: string
+          required: true
+      service-account:
```

Comment:
We have to move this parameter to secrets. 
It should not be required.
Let's use specific names like `wid-token-service-account` and change the description to `The service account of the requestor service. Will be used during Workload Identity token generation.`.